src/tasks/MonoTargetsTasks/ILStrip/AssemblyStripper/AssemblyStripper.cs

```diff
@@ -237,5 +237,19 @@ public static void StripAssembly(string assemblyFile, string outputPath)
             AssemblyDefinition assembly = AssemblyFactory.GetAssembly(assemblyFile);
             AssemblyStripper.StripAssembly(assembly, outputPath);
         }
+
+        public static bool TryStripAssembly(string assemblyFile, string outputPath)
+        {
+            try
+            {
+                StripAssembly(assemblyFile, outputPath);
+                return true;
+            }
+            // Skip unmanged assemblies
+            catch (ImageFormatException)
+            {
+                return false;
+            }
```

Comment:
It might be helpful to catch general failures and report them as well.

Reply:
The caller of this function has a try catch block reporting all the other failures.